code/modules/mob/living/carbon/human/human_defense.dm

```diff
@@ -76,6 +94,21 @@ emp_act
 		protection += M.rad_protection
 	return protection
 
+/mob/living/carbon/human/proc/checkarmorabsorb(var/datum/organ/external/def_zone, var/type)
+	if(!type)
+		return 0
+	var/protection = 0
+	var/list/body_parts = list(head, wear_mask, wear_suit, w_uniform, gloves, shoes)
+	for(var/bp in body_parts)
+		if(!bp)
+			continue
+		if(bp && istype(bp ,/obj/item/clothing))
```

Comment:
at least the first check on the this line is redundant, you could also get rid of the one above because `istype` will fail for null values. Also dat comma.

Reply:
Yeah, both the `!bp` and `bp &&` here are redundant. Remove them please. And fix that comma.